scrapers/HandDomination.yml

```
@@ -0,0 +1,34 @@
+name: HandDomination
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - handdomination.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h3[@class="videoTitle"]
+      Details:
+        selector: //li[@class="videoDescription"]/p
+      Tags:
+       Name: //li[@class="videoCategories"]/p/a
+      Date:
+        selector: //li[@class="update_posted"]/p
+        postProcess:
+          - replace:
+              - regex: "date:"
```

I would replace  the spaces after the `date:` just to be safe and not rely on stash's trimming.
```suggestion
              - regex: 'date:\s+'
```